app/controllers/api/functions.php

```diff
@@ -769,6 +769,16 @@
             throw new Exception('Deployment not found. Deploy deployment before trying to execute a function', 404);
```

Comment:
Deploy a deployment sound weird, can we switch to create a deployment?

Reply:
I will update it 👍